lesson6.html

```diff
@@ -47,12 +47,13 @@ <h3>Dictionaries</h3>
 
             <div class="fragment">
               <pre><code class="python" data-trim>
-                eng_to_sp = {} # create an empty dictionary
+                eng_dict = {} # let's create an English dictionary
```

Comment:
Would it be good to name this as `english_dictionary`? It doesn't take long to type, but it reinforces the idea that variables should be well-named.

Reply:
The only reason I put this is... so it fits in one line in the code block below. I'll try and expand the code block somehow!

Reply:
Ah, then it's fine like that 😄